### ISSUE TITLE
Anti-Potassium channel tetramerization domain-containing (KCTD)16 antibody

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10545]
**Preferred term label:**

**Synonyms**
Anti-KCTD16

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against Potassium channel tetramerization domain-containing (KCTD)16.

PMID: 31009048 (....) Anti-GABABR encephalitis is a limbic encephalitis with prominent, severe seizures, but patients can also present with rapidly progressive dementia. The co-occurrence of KCTD16 antibodies points towards a paraneoplastic origin. The addition of KCTD16 improves the sensitivity of the cell-based assay

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:624166 Non-specific autoimmune supratentorial encephalitis with characteristic antibodies

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000878
- label: Anti-potassium channel tetramerization domain containing 16 antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:31009048
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against potassium channel tetramerization domain containing 16 (KCTD16).
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10545
- synonyms: Anti-KCTD16(NONE)
